lib/ndr_parquet_generator.rb

```diff
@@ -99,7 +99,11 @@ def arrow_field_types(table)
           column['mappings'].each do |mapping|
             field = mapping['field']
             arrow_data_type = mapping['arrow_data_type'] || :string
-            field_types[klass][field] = arrow_data_type
+            if arrow_data_type == :list
+              field_types[klass][field] = mapping['arrow_list_field'].symbolize_keys
```

Comment:
Might you want to “fetch” the value here, so if it isn’t provided you get a helpful error?

Reply:
Thanks @joshpencheon - 3b4c769